qt/widgets/common/CMakeLists.txt

```diff
@@ -55,7 +55,7 @@ set(
   src/Message.cpp
   src/MessageDisplay.cpp
   src/MultifitSetupDialog.cpp
-  src/MWRunFiles.cpp
+  src/FileFinderWidget.cpp
```

Comment:
Can you sort these lists alphabetically?

Reply:
Done